recipes/spy/0.0.3/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+cmake_minimum_required(VERSION 2.8.12)
+project(PackageTest CXX)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup()
+
+add_library(spy_config INTERFACE)
+target_compile_features ( spy_config INTERFACE  cxx_std_17 )
+
+add_executable(example example.cpp)
+target_link_libraries(example ${CONAN_LIBS} PUBLIC spy_config)
```

Comment:
```suggestion
add_executable(example example.cpp)
target_link_libraries(example ${CONAN_LIBS})
set_property(TARGET example PROPERTY CXX_STANDARD 17)
```

keep it simple

Reply:
Well, the use of cxx_std_17 is the proper CMake way to pass standard to target as the latest CMake doc advises.

Reply:
target_compile_features is CMake 3+ Almost all official recipes use this approach.